src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, {useState} from "react";
+import {TaskButton} from "../../Atoms/AddTaskButton";
+import {Task} from "../../Molecules/Task";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+
+export const TodoCard = () => {
+
+    const [taskList,setTaskList] = useState([]);
+
+
+    const onAddTaskButtonClick = () => {
+        //Spread構文: [...array]で配列を展開 ＝＞ 配列末尾にオブジェクトを追加
+        setTaskList(taskList => [...taskList, {name:"", initializing: true}]);
```

Comment:
```suggestion
        setTaskList([...taskList, {name:"", initializing: true}]);
```
このように書いても同じ意味になります！（挙動に影響はないため、修正するかどうかはお任せします！）